api/swagger.yaml

```diff
@@ -7958,6 +7958,9 @@ paths:
           description: |
             A JSON encoded value of the filters (a `map[string][]string`) to process on the secrets list. Available filters:
 
+            - `id=<secret id>`
+            - `label=<key> or label=<key>=value`
+            - `name=<secret name>`
             - `names=<secret name>`
```

Comment:
Sorry for my delay. I am afraid that currently we support `names`. And this is something we hope to deprecate to use `name` instead. Actually it is indeed supported in docker/master.